src/components/WhoWeAre/Departments.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+//import Row from 'react-bootstrap/Row';
+//import Col from 'react-bootstrap/Col';
+import Accordion from 'react-bootstrap/Accordion';
+
+import Bios from './Bios';
+
+const Department = () => {
+  return (
+    <AccordionSection className="container">
+      <Accordion defaultActiveKey="0">
+        <Accordion.Toggle className="accordionHeader row" eventKey="0">
+          <h2>test</h2>
+        </Accordion.Toggle>
+        <Accordion.Collapse eventKey="0">
+          <p>SOME Text</p>
+        </Accordion.Collapse>
+
+        <Accordion.Toggle className="accordionHeader row" eventKey="1">
+          <h2>ADVISORY BOARD</h2>
+        </Accordion.Toggle>
+        <Accordion.Collapse eventKey="1">
+          <p>SOME Text</p>
+        </Accordion.Collapse>
+
+        {console.log(Bios)}
```

Comment:
So you'll import bios above `import bios from './bios';` and then you'll map it here. For example:
```
{bios.map((bio, i) => (
    <React.Fragment key={bio.name}>
        <Accordion.Toggle className="accordionHeader row" eventKey={2 + i}>
            <h2>{bio.name}</h2>
        </Accordion.Toggle>
        <Accordion.Collapse eventKey="1">
          <img alt={bio.name} src={image} />
          <h3>{bio.pronouns}</h3>
          <p>{bio.bio}</p>
        </Accordion.Collapse>
    </React.Fragment>
)}
```